### PR TITLE
feat: improve nb_NO translations

### PR DESCRIPTION
**Pull request by Johannes-Andersen**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [X] Other: **Localization improvements**

### 🔗 Related issue link

No tickets

### 💡 Background and solution

There was a huge lack of translations for nb_NO. I have updated some mistakes from rc-pagination and added translations values from en_US to the NO templates where missing

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | 🇳🇴  Added missing nb_NO translations  |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by Johannes-Andersen** [comment]
This is my first pull request. So feel free to be as be critical and let me know if there is something wrong or something else needed/missing. 

Thanks,
Johannes

File: components/date-picker/locale/nb_NO.tsx [hunk 1]
```diff
@@ -1,17 +1,53 @@
-import CalendarLocale from 'rc-picker/lib/locale/nb_NO';
```
Comment (yoyo837) [inline components/date-picker/locale/nb_NO.tsx:1]: Why did you remove this?　You need to send PR to rc-picker　at　first

Comment (Johannes-Andersen) [reply components/date-picker/locale/nb_NO.tsx:1]: Thanks for the quick reply. Create a pull there, https://github.com/react-component/picker/pull/97

Will update diff when/if thats merged and released

Comment (yoyo837) [reply components/date-picker/locale/nb_NO.tsx:1]: Waiting a new patch.

Comment (yoyo837) [reply components/date-picker/locale/nb_NO.tsx:1]: rc-picker v1.10.6, Pls remove `node_modules` and `package-lock.json` or `yarn.lock` then reinstall it.

File: components/date-picker/locale/nb_NO.tsx [hunk 2]
```diff
@@ -1,17 +1,53 @@
...
 const locale: PickerLocale = {
   lang: {
+    locale: 'nb_NO',
     placeholder: 'Velg dato',
+    yearPlaceholder: 'Velg år',
+    quarterPlaceholder: 'Velg kvartal',
+    monthPlaceholder: 'Velg måned',
+    weekPlaceholder: 'Velg uke',
     rangePlaceholder: ['Startdato', 'Sluttdato'],
-    ...CalendarLocale,
```
Comment (yoyo837) [inline components/date-picker/locale/nb_NO.tsx:10]: revert this

Comment (Johannes-Andersen) [reply components/date-picker/locale/nb_NO.tsx:10]: Updated. All the placeholders are not in rc-picker, but is in the other translations. So keeping them unless you think they also should go
